kafka/integration_test.go

```diff
@@ -337,12 +338,16 @@ func consumerTest(t *testing.T, testname string, msgcnt int, cc consumerCtrl, co
 
 	conf := ConfigMap{"bootstrap.servers": testconf.Brokers,
 		"go.events.channel.enable": cc.useChannel,
-		"group.id":                 testconf.GroupID,
-		"session.timeout.ms":       6000,
-		"api.version.request":      "true",
-		"enable.auto.commit":       cc.autoCommit,
-		"debug":                    ",",
-		"auto.offset.reset":        "earliest"}
+		"group.id": testconf.GroupID +
+			fmt.Sprintf("-%d", rand.Intn(1000000)),
```

Comment:
nit: a 1/Uuid chance of dying would be better

Reply:
There's no uuid generator in the standard library, don't want to pull in an external dependency just for this.